fdbserver/CommitProxyServer.actor.cpp

```diff
@@ -2345,8 +2345,9 @@ ACTOR Future<Void> postResolution(CommitBatchContext* self) {
 	// If a transaction has any mutation accessing to the locked range, reject the transaction with
 	// error_code_transaction_rejected_range_locked
 	// This feature is disabled when the database is locked
```

Comment:
Would you like to extend this comment and mention the other features that the range locks are incompatible with?

Reply:
Not be able to commit to this PR. So, address the comments in a separate PR https://github.com/apple/foundationdb/pull/11748